### ISSUE TITLE
What kind(s) of permanent identifiers?

### ISSUE THREAD

**Issue by kintopp** [culturesofknowledge/emplaces#6]
Placeholder for a discussion on how and what kind of identifiers to support in EM Places (also with a view towards reusing this system in EM People, and possibly EM Dates). See:

ARK info: http://n2t.net/e/ark_ids.html (n2t.net is the resolver service) and DOI CataCite info: https://schema.datacite.org/meta/kernel-4.0/

**Comment by kintopp** [comment]
UUIDs are awkwardly long for purposes of academic citation (footnotes, bibliography..). But let's see what the Huygens suggests.

**Comment by gklyne** [comment]
We also need to think about longevity for any ontology/vocabulary terms we introduce (the "em:" prefixes in the Opole sample data.  PURL is a traditional solution here, but I'm, not sure about the status of purl.org these days.

UUIDs are fine for stuff that's never seen by a human, but I agree they're cumbersome for human interaction.

I think that we need to make our own call on persistent URIs.

Every new domain name we use is one for which registrar fees should be paid for as long as we want the names to remain valid.  (Even if they aren't usually dereferenced, I think there should be some attempt to provide useful information through them, supporting the web "follow your nose" principle.)

**Comment by marnixvb** [comment]
We will build redirect functionality into Timbuctoo, so a PID can be 'minted' to a subject (as default or by user choice). That would be able to communicate with your identifier service of choice -- we currently use handles. We assume Oxford will host its own identifier service.

Issue to discuss:
- will the identifier redirect to a specific version of the subject 
OR
- will the identifier redirect to the latest version, where all other versions can be navigated ?

**Comment by neilsjefferies** [comment]
At Oxford we currently use UUID's for individual objects (mostly, there is a history of random accession numbers too) but are looking at using ARK's for "things" (so a catalogue entry for an item, the item and the digtitised images may well have different UUID's but can be referenced using the same ARK. We plan to update our PURL resolver to handle the ARK's plus also push to N2T. We are also looking at issuing ARK's for college owned materials so we will support (in some manner tbd) the concept of subdomains which would fit EMLO's uses. Versioning will be supported (ARK let's us do this).

**Comment by kintopp** [comment]
Neil, what's the (conservative) timeline at the library for supporting ARKs?

**Comment by neilsjefferies** [comment]
We're already signed up and should have the requisite server 
infrastructure set up mid-August.

N

**Comment by kintopp** [comment]
Thanks. And reading over http://n2t.net/e/ark_ids.html have I understood correctly that we could then publish an ARK in the form:  http://emplaces.info/ark:/13030/tf5p30086k? i.e. that the NMA portion of the ARK is swappable by us to whatever we like?

**Comment by neilsjefferies** [comment]
Yes, we can mint arks which can be used elsewhere - we can thus use the 
same identifier for the Oxford and EMwhatever - which just makes life 
easier.

N
